jobs/ci-kubernetes-kubemark-gce-scale.env

```diff
@@ -3,7 +3,7 @@
 PROJECT=kubernetes-scale
 USE_KUBEMARK=true
 KUBEMARK_TESTS=\[Feature:Performance\]
-KUBEMARK_TEST_ARGS=--gather-resource-usage=true --garbage-collector-enabled=true --kube-api-content-type=application/vnd.kubernetes.protobuf
+KUBEMARK_TEST_ARGS=--gather-resource-usage=true --garbage-collector-enabled=true --kube-api-content-type=application/vnd.kubernetes.protobuf --output-print-type=json
```

Comment:
And here also remove "--kube-api-content-type" this is also now defaulted to protobuf.